lib/matplotlib/backends/backend_ps.py

```diff
@@ -981,8 +975,7 @@ def _print_figure(
         the key 'Creator' is used.
         """
         isEPSF = format == 'eps'
-        if isinstance(outfile,
-                      (six.string_types, getattr(os, "PathLike", ()),)):
+        if isinstance(outfile, (str, getattr(os, "PathLike", ()),)):
```

Comment:
This `getattr` looks like another back-compat shim, how far back do we need this?

Reply:
`os.PathLike` is new in Python 3.6, so it will be needed for a while yet.